Geospatial.md

```diff
@@ -94,6 +94,41 @@ Bounding box is defined as the thrift struct below in the representation of
 min/max value pair of coordinates from each axis. Note that X and Y Values are
 always present. Z and M are omitted for 2D geospatial instances.
 
+Writers should follow the guidelines below when calculating bounding boxes in
+the presence of edge cases.
+
+* `null` instance: Skip it and continue processing the remaining 
+  geospatial instances. Do not produce a bounding box if all instances are null.
+* Non-`null` instance with [special geospatial values](#special-geospatial-values):
+  * X and Y: Skip any special X or Y value and continue processing the 
+    remaining X or Y values. Do not produce a bounding box if all X or all Y 
+    values are special values.
+
+  * Z: Skip any special Z value and continue processing the remaining Z values.
```

Comment:
nit: should we just omit 'continue processing the remaining...' in all these sections, to reduce the verbosity , as its a bit obvious?

Reply:
If you don't have a strong opinion here, I kind of want to keep it verbose. We tend to make it clear that one should only omit individual X/Y/Z/M value and continue using the remaining values, instead of dropping the coordinate or whole geospatial instance.

Reply:
ok i think its fine with me.  For me It'd probably be more helpful if somehow we had that context (talking about continuing with rest of coordinates in an instance), than it is now.  Thinking if we can put this somewhere, but agree not in this line though.